cmd/main.go

```diff
@@ -188,3 +191,7 @@ func getProjectVersion() (bool, string) {
 	}
 	return true, projectInfo.Version
 }
+
+func printV1DeprecationWarning() {
+	fmt.Printf("warning: [deprecation] v1 projects are deprecated and will not be supported beyond Feb 1, 2020. Refer to https://book.kubebuilder.io/migration/guide.html for instructions on how to migrate to v2.\n")
```

Comment:
```suggestion
	log.Printf(NoticeColor,"[Deprecation Notice] - The v1 projects are deprecated and will not be supported beyond Feb 1, 2020.\n See how to upgrade your project to v2: https://book.kubebuilder.io/migration/guide.html\n")
```
For it noticed ;-) WDYT?

<img width="1119" alt="Screenshot 2019-11-07 at 00 58 51" src="https://user-images.githubusercontent.com/7708031/68350950-ebbb5180-00f9-11ea-8639-d4ebda52cc76.png">

Reply:
Done